docs/advanced-topics/networktime-ticks.md

```diff
@@ -0,0 +1,204 @@
+    }
+}
+```
+
+### Example 2: Using network time to create a synced event
+
+Most of the time aligning an effect precisely to time is not needed. But in some cases for important effects or gameplay events it can help to improve consistency especially for clients with bad network connections.
+
+```cs
+using System.Collections;
+using Unity.Netcode;
+using UnityEngine;
+using UnityEngine.Assertions;
+
+public class SyncedEventExample : NetworkBehaviour
+{
+    public GameObject ParticleEffect;
+
+    // Called by the client to create a synced particle event at its own position.
+    public void ClientCreateSyncedEffect()
+    {
+        Assert.IsTrue(IsOwner);
+        var time = NetworkManager.LocalTime.Time;
+        CreateSyncedEffectServerRpc(time);
+        StartCoroutine(WaitAndSpawnSyncedEffect(0)); // Create the effect immediately locally.
+    }
+
+    private IEnumerator WaitAndSpawnSyncedEffect(float timeToWait)
+    {
+        // Note sometimes the timeToWait will be negative on the server or the receiving clients if a message got delayed by the network for a long time. This usually happens only in rare cases. Custom logic could be implemented to deal with that scenario.
+        if (timeToWait > 0)
+        {
+            yield return new WaitForSeconds(timeToWait);
+        }
+
+        Instantiate(ParticleEffect, transform.position, Quaternion.identity);
+    }
+
+    [ServerRpc]
+    private void CreateSyncedEffectServerRpc(double time)
+    {
+        CreateSyncedEffectClientRpc(time); // Call a client RPC to also create the effect on each client.
+        var timeToWait = time - NetworkManager.ServerTime.Time;
+        StartCoroutine(WaitAndSpawnSyncedEffect((float)timeToWait)); // Create the effect on the server but wait for the right time.
+    }
+
+    [ClientRpc]
+    private void CreateSyncedEffectClientRpc(double time)
+    {
+        // The owner already created the effect so skip them.
+        if (IsOwner == false)
+        {
+            var timeToWait = time - NetworkManager.ServerTime.Time;
+            StartCoroutine(WaitAndSpawnSyncedEffect((float)timeToWait)); // Create the effect on the client but wait for the right time.
+        }
+    }
+}
+```
+
+<Mermaid chart={`
+sequenceDiagram
+    participant Owner as Owner
+    participant Server as Server
+    participant Receiver as Other Client
+
+    Note over Owner: LocalTime = 10.0
+    Note over Owner: ClientCreateSyncedEffect()
+    Note over Owner: Instantiate effect immediately (LocalTime = 10)
+    Owner->>Server: CreateSyncedEffectServerRpc
+
+    Server->>Receiver: CreateSyncedEffectClientRpc
+
+    Note over Server: ServerTime = 9.95 #38; timeToWait = 0.05
+    Note over Server: StartCoroutine(WaitAndSpawnSyncedEffect(0.05))
+    Server->>Server: WaitForSeconds(0.05);
+    Note over Server: Instantiate effect at ServerTime = 10.0
+
+    Note over Receiver: ServerTime = 9.93 #38; timeToWait = 0.07
```

Comment:
I really wanted this part to happen in parallel to the block above but I couldn't figure out how with mermaid